src/features/command-palette/server-command-palette-provider.ts

```diff
@@ -28,15 +29,15 @@ export namespace ServerCommandPalette {
 @injectable()
 export class ServerCommandPaletteActionProvider implements ICommandPaletteActionProvider {
 
-    constructor(@inject(TYPES.IActionDispatcher) protected actionDispatcher: GLSPActionDispatcher) { }
+    constructor(@inject(TYPES.IActionDispatcher) protected actionDispatcher: GLSPActionDispatcher,
```

Comment:
Is there a particular reason to use constructor injection here? Otherwise I'd suggest to refactor this to property injection (easier to extend)